src/components/Annotation/Annotation.scss

```diff
@@ -107,3 +93,40 @@
         font-size: inherit;
     }
 }
+
+.ba-mobile-annotation-dialog .ba-annotation {
```

Comment:
Is this currently unused?

Reply:
`.ba-mobile-annotation-dialog` is applied when annotations is used on smaller screen sizes so this CSS does get used